image/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM registry.access.redhat.com/ubi8/ubi
+
+RUN curl -L https://github.com/hacbs-contract/ec-cli/releases/download/snapshot/ec_linux_amd64 -o /usr/bin/ec && chmod +x /usr/bin/ec && ec version
```

Comment:
Install anything besides ec?

Reply:
I don't think we need anything else in here.

If we put this in the [release workflow](https://github.com/hacbs-contract/ec-cli/blob/9b3412a103f0ec3b70cb4ceaeb4750ffdcac85e8/.github/workflows/release.yaml), we don't need to fetch the snapshot release, but copy from the `dist/`. Also helps with determinism, this will always build an image with a snapshot version whereas release workflow will build with the version for a specific commit.